### ISSUE TITLE
Fotmob issue

### ISSUE THREAD

**Issue by mbrownsword** [JaseZiv/worldfootballR#262]
Not sure if this is related to an issue mentioned previously, but with this code, I'm getting:

```
fotmob_get_league_matches(league_id = id, cached = FALSE)
Error in open.connection(x, "rb") : HTTP error 403.
```

**Comment by mbrownsword** [comment]
Did Fotmob block you?

**Comment by mbrownsword** [comment]
I think we all got blocked from accessing their website @tonyelhabr @JaseZiv

**Comment by mbrownsword** [comment]
sorry for the stream of conciousness but how does fotmob_get_league_ids work and not fotmob_get_league_matches?

**Comment by tonyelhabr** [comment]
yes, i can confirm that this is not working. my guess is that they have blocked the user agent to the endpoint for this function. i have found a way to get around this, but i'll have to think about how to implement it

**Comment by mbrownsword** [comment]
thank you!

**Comment by tonyelhabr** [comment]
addressed with #263
